doc/user/content/sql/create-source/webhook.md

```diff
@@ -64,7 +64,15 @@ Column     | Type                        | Optional?
 
 ## Webhook URL
 
-After creating a webhook source, send **POST** requests to `https://<HOST>/api/webhook/<database>/<schema>/<src_name>`.
+The URL which can be used to **POST** events to your webhook source is stored in the
+[`mz_internal.mz_webhook_sources`](/sql/system-catalog/mz_internal/#mz_webhook_sources) system table.
```

Comment:
Is internal tables where we usually keep this kind of information? I would expect that we have a non-internal place to expose this (a SHOW command?) if we are documenting that users need to run it. I'm ok keeping it if this is what we do in other places.

Reply:
Generally we want to put a table like this in mz_catalog, but since webhooks are still in private preview, and we might want to iterate on this table, we've opted to keep it in mz_internal for now so we can maintain flexibility